C/plugins/filter/python27/plugin.cpp

```diff
@@ -0,0 +1,602 @@
+/*
+ * FogLAMP "Python 2.7" filter plugin.
+ *
+ * Copyright (c) 2018 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Massimiliano Pinto
+ */
+
+#include <plugin_api.h>
+#include <config_category.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <strings.h>
+#include <string>
+#include <iostream>
+#include <filter_plugin.h>
+#include <filter.h>
+#include <reading_set.h>
+
+#include <Python.h>
+
+#define FOGLAMP_ROOT_PATH    "/usr/local/foglamp"
```

Comment:
We do this stuff to find FOGLAMP_ROOT all over the place, we really should have a common place for it.

Reply:
Yes, FOGL-1699 will take care of this